rules/windows/defense_evasion_suspicious_process_access_direct_syscall.toml

```diff
@@ -35,7 +35,22 @@ process where event.code == "10" and
  length(winlog.event_data.CallTrace) > 0 and
 
  /* Sysmon CallTrace starting with unknown memory module instead of ntdll which host Windows NT Syscalls */
- not winlog.event_data.CallTrace : ("?:\\WINDOWS\\SYSTEM32\\ntdll.dll*", "?:\\WINDOWS\\SysWOW64\\ntdll.dll*")
+ not winlog.event_data.CallTrace :
+            ("?:\\WINDOWS\\SYSTEM32\\ntdll.dll*",
+             "?:\\WINDOWS\\SysWOW64\\ntdll.dll*",
+             "?:\\Windows\\System32\\wow64cpu.dll*",
+             "?:\\WINDOWS\\System32\\wow64win.dll*",
+             "?:\\Windows\\System32\\win32u.dll*") and
+
+ not winlog.event_data.TargetImage :
+            ("?:\\Program Files (x86)\\Malwarebytes Anti-Exploit\\mbae-svc.exe",
+             "?:\\Program Files\\Cisco\\AMP\\*\\sfc.exe",
+             "?:\\Program Files (x86)\\Microsoft\\EdgeWebView\\Application\\*\\msedgewebview2.exe",
+             "?:\\Program Files\\Adobe\\Acrobat DC\\Acrobat\\*\\AcroCEF.exe") and
+
+ not (process.executable : ("?:\\Program Files\\Adobe\\Acrobat DC\\Acrobat\\Acrobat.exe",
+                            "?:\\Program Files (x86)\\World of Warcraft\\_classic_\\WowClassic.exe") and
```

Comment:
May need to look into ignoring the signer `Blizzard Entertainment, Inc.` for this. Video games can be launched from various directories depending on where they are installed. Some install them in various drives, for storage purposes, and the executable location could be anywhere. Additionally, with World of Warcraft you may use a blizzard launcher or launcher specifically for the game so there are several executables we would have to account for, whereas the signer may allow us to just ignore it in general. This executable only accounts for a single version of the game.

Reply:
there is no codesign metadata in sysmon event 10 (Process Access) .